src/app.ts

```diff
@@ -64,29 +69,64 @@ async function app() {
       'exit with error if this is not the latest version of SUSHI',
       false
     )
-    .option('-i, --init', 'initialize a SUSHI project')
-    .option('-u, --update-dependencies', 'update FHIR packages in project configuration')
-    .version(getVersion(), '-v, --version', 'print SUSHI version')
+    .action(async function (projectPath, options) {
+      await runBuild(projectPath, options);
+    })
     .on('--help', () => {
       console.log('');
       console.log('Additional information:');
       console.log('  [path-to-fsh-project]');
       console.log('    Default: "."');
       console.log('  -o, --out <out>');
       console.log('    Default: "fsh-generated"');
+    });
+
+  program
+    .command('updateDependencies [path-to-fsh-project]')
+    .description('update FHIR packages in project configuration')
+    .action(async function (projectPath) {
+      const input = ensureInputDir(projectPath);
+      const config: Configuration = readConfig(input);
+      await updateExternalDependencies(config);
+      process.exit(0);
     })
-    .arguments('[path-to-fsh-defs]')
-    .action(function (pathToFshDefs) {
-      input = pathToFshDefs;
-    })
-    .showHelpAfterError()
-    .parse(process.argv)
-    .opts();
+    .on('--help', () => {
+      console.log('');
+      console.log('Additional information:');
+      console.log('  [path-to-fsh-project]');
+      console.log('    Default: "."');
+    });
 
-  if (program.init) {
-    await init();
-    process.exit(0);
+  program
+    .command('init')
```

Comment:
When you run `sushi help`, it prints these out in the order we have them in the file.  So, right now, it shows:
```
Commands:
  build [options] [path-to-fsh-project]     export a FSH project into a FHIR IG
  updateDependencies [path-to-fsh-project]  update FHIR packages in project configuration
  init                                      initialize a SUSHI project
  help [command]                            display help for command
```

But (for whatever reason), I think I would prefer:
```
Commands:
  build [options] [path-to-fsh-project]      build a sushi project
  init                                      initialize a SUSHI project
  update-dependencies [path-to-fsh-project]  update FHIR packages in project configuration
  help [command]                            display help for command
```
Maybe because it is alphabetical (with `help` at the end)? Or maybe because `init` seems more useful than `update-dependencies`?  I dunno.  What do you think?

If you think you like `init` before `update-dependencies` too, then you'll need to move this block of code up.

Reply:
I also like it, both for alphabetical order but also because I agree, `init` seems more useful. I'll move it up.

Reply:
See: [50da4cd](https://github.com/FHIR/sushi/pull/1163/commits/50da4cd43388038d724ac58ebcb4f622e3d533b2)